src/resolver.rs

```diff
@@ -69,18 +69,20 @@ impl<'ctx> DependencyResolver<'ctx> {
         // Store path dependencies already in checkout_dir
         match self.sess.manifest.workspace.checkout_dir.clone() {
             Some(checkout) => {
-                for dir in fs::read_dir(checkout).unwrap() {
-                    self.checked_out.insert(
-                        dir.as_ref()
-                            .unwrap()
-                            .path()
-                            .file_name()
-                            .unwrap()
-                            .to_str()
-                            .unwrap()
-                            .to_string(),
-                        config::Dependency::Path(dir.unwrap().path()),
-                    );
+                if checkout.exists() {
```

Comment:
If this is not true, no `checkout_dir` will be used, correct? Is this what we want, or would we want Bender to (try to) create the checkout dir in that case?

Reply:
The creation and filling of the `checkout_dir` happens at a later stage, the code here only checks for dependencies already there. This appropriately updates other dependencies based on changes to the manifest within these folders (e.g. adding a new dependency to the `Bender.yml`), but is mot needed for the first step of checking out the files.

Reply:
I see, thanks for the explanation!